src/system/ZAuthModule/Container/LinkContainer.php

```diff
@@ -148,8 +148,8 @@ private function getUser()
         $links = $this->getAccount();
         array_unshift($links, [
             'url'   => $this->router->generate('zikulausersmodule_account_menu'),
-            'text' => $this->translator->__('Menu'),
-            'icon'  => 'cubes'
+            'text' => $this->translator->__('Account menu'),
+            'icon'  => 'fa fa-user-circle'
```

Comment:
the other is `fa fa-user-circle-o` and this is `fa fa-user-circle`. I thought you wanted them the same?